horizon/structs/doc.go

```diff
@@ -0,0 +1,2 @@
+// Package structs contains response structs from horizon
+package structs
```

Comment:
`structs` is IMO not a good name... it doesn't communicate anything to the reader.  In my experience, many RESTful APIs refer to the types used in their responses as "Resources", and that's what I've also done on the horizon server side.  I would recommend you use something like `resource`, `res`, or `response` instead.